src/features/scheduled-messages.ts

```diff
@@ -1,15 +1,35 @@
-import { MessageOptions } from "child_process";
-import * as discord from "discord.js";
+import type * as discord from "discord.js";
 import cron from "node-cron";
+import { logger } from "./log";
 
+const enum CHANNELS {
+  "helpReact" = "103696749012467712",
+  "helpJs" = "565213527673929729",
+  "random" = "103325358643752960",
+  "jobBoard" = "103882387330457600",
+}
+
+// By keeping these at odd divisions, we can make sure they show up at all timezones. If it were */24, for instance, it would consistently show up in the middle of the night for some timezones.
+const enum FREQUENCY {
```

Comment:
Instead of cron, we should swap this out to be a zeroed-out timer instead. On bot startup, init relative to Sunday midnight and configure intervals against that. cron doesn't have a way of doing cross-day time durations, like sending a message every 18 hours.